packages/create-react-app/index.js

```diff
@@ -130,7 +130,7 @@ function createApp(name, verbose, version) {
   process.chdir(root);
 
   console.log('Installing packages. This might take a couple minutes.');
-  console.log('Installing ' + chalk.cyan('react-scripts') + '...');
+  console.log('Installing ' + chalk.cyan('react, react-dom, react-scripts') + '...');
```

Comment:
@fson 

I just noticed that we allow user set custom packageName via --scripts-version. better we move this lines to install function and replace with `react-scripts` with `packageName`. What do you think?